cypress/pageObjects/AdminPortal/AdminEventPage.ts

```
@@ -147,4 +141,4 @@ export class AdminEventPage {
     cy.contains(this._eventCard, eventTitle, { timeout }).should('not.exist');
     return this;
   }
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)